cmd/exporter.go

```diff
@@ -31,6 +30,8 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/prometheus/common/version"
+
+	"k8s.io/klog/v2"
```

Comment:
does klog pkg exist in go.mod and vendor pkg?

Reply:
@rootfs updated!